exercises/modules/modules2.cairo

```diff
@@ -0,0 +1,56 @@
+// I AM NOT DONE
+// These modules have some issues, can you fix?
+// Run `starklings hint modules2` or `hint` watch command for a hint.
+
+use debug::PrintTrait;
+const YEAR: felt252 = 2050;
```

Comment:
let's avoid using felt252 where possible - we want to push good practices, a year fits in a u16 so we should use a u16 here